### PR TITLE
Add device type list to readme

### PR DESCRIPTION
**Pull request by Apollon77**
I collected from best knowledge what I found compatible ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,18 +106,64 @@ Completed implementations:
...
+## Device types supported by Ecosystems
+
+This list is a compilation of published information and own community tests. Many updates in the ecosystems happen without big announcements, so the information here can be a bit outdated and more device types are supported. If you find errors or added device types please open a PR or issue to report them. Especially for SmartThings and Tuya no information is officially published. 
+
+Table Legend:
+* "X" means supported
+* "-" means not supported from current knowledge
+* "?" means unknown
+
+| Gerätetyp                         | Apple | Google        | Amazon | SmartThings | Tuya |
```
Comment (lauckhart) [inline README.md:122]: Does "Gerätetyp" = "feature"?

**Review by lauckhart — APPROVED** [review]
I think this is fine for now but it would be cool to get this type of information into a machine-readable format.  We could then generate documentation but also use programmatically to create warnings and/or tailor published devices for specific clients.

**Review by JimBuzbee — APPROVED** [review]
Did a bit of testing with SmartThings. As usual, it's a bit of a pain to get SmartThings to pair, but I was able to get the following Matter devices working: Contact, Light, Temperature, Humidity, and Flow

File: README.md (hunk 1 above)
Comment (Apollon77) [reply README.md:122]: Devcie type ... (sorry german leftover, fixed)

**Pull request merged**